source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,485 @@
+  [ADD REF LATER].
+- Versions. This page lists the Edge Server's current version, and the latest
+  available version. If the Edge Server falls more than one major version 
+  behind the latest available, it enters a :guilabel:`Failed` state and cannot
+  sync with Atlas until it is upgraded. For more information, refer to 
+  :ref:`upgrade-edge-server-version`.
+- The Install command that you can use in the terminal to install the 
+  Edge Server code.
+- A button to regenerate the registration token.
+- A button to delete the Edge Server instance.
+
+.. _delete-edge-server-instance:
+
+Delete an Edge Server Instance
+------------------------------
+
+You can delete an individual Edge Server instance from your Edge Server Service.
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. procedure::
+
+         .. step:: Browse to the Edge Server Dashboard
+
+            Click the :guilabel:`Edge Server` option in the left navigation 
+            sidebar to open the Edge Server dashboard.
+
+         .. step:: View the Edge Server Instance Details
+
+            Find the Edge Server instance you want to delete in the list,
+            and click the :guilabel:`View Details` button.
+
+         .. step:: Delete the Edge Server Instance
+
+            Click the :guilabel:`Delete` button at the bottom of the Edge
+            Server details view.
+
+            You'll be prompted to confirm deletion. Follow the prompts to
+            confirm that you want to delete the Edge Server.
+
+   .. tab::
+      :tabid: api
+
+      .. procedure::
+
+         .. step:: Authenticate a MongoDB Atlas User
+
+            .. include:: /includes/api-authenticate-instructions.rst
+
+         .. step:: Get the ID of the Edge Server to Delete
+
+            Call the <LIST EDGE SERVER> endpoint to get a list of all the 
+            Edge Servers instances:
+
+            .. code-block:: shell
+
+               INSERT API DETAILS HERE
+
+            Find the ID of the Edge Server that you want to delete in the list.
+
+         .. step:: Call the Delete Endpoint
+
+            Call the <DELETE EDGE SERVER> endpoint with the ID of the Edge
+            Server instance that you want to delete.
+
+            .. code-block:: shell
+
+               INSERT API DETAILS HERE
+
+.. _update-edge-server-data-model:
+
+Update an Edge Server's Data Model
+----------------------------------
+
+The process to update a data model varies depending on whether you're using
+an Edge Server Service or a Device Sync Service with Edge Server.
+
+Edge Server Service
+~~~~~~~~~~~~~~~~~~~
+
+When you're using an Edge Server Service, the service automatically updates
+properties in a collection when you make changes to the documents you sync.
+However, some types of changes cause an Edge Server instance to reboot and
+wipe itself.
+
+If you're adding a new collection you haven't synced before, you must define
+a new schema that contains only the collection's name.
```

Comment:
😬 They will also need to define the _id field as an ObjectID. On that note, I think we need to clarify somewhere that the Edge Server service currently only supports documents with _id fields that are ObjectIDs, other types are not currently supported (again this is only for the Edge Server service).